offix/src/main/java/org/aerogear/offix/interceptor/ConflictInterceptor.kt

```diff
@@ -40,6 +49,7 @@ class ConflictInterceptor(private val conflictResolutionImpl: ConfliceResolution
     inner class OffixConflictCallback(val conflictResolutionImpl: ConfliceResolutionInterface) :
         ApolloInterceptor.CallBack {
         private val TAG = javaClass.simpleName
+        val userCallback = queueCallback.removeFirst()
```

Comment:
You forgot using the userCallback in onResponse and onFailure.
No need to rush, take your time to implement and check this before pushing.

Reply:
Umm, I have used it.